daemon/container_operations.go

```diff
@@ -118,6 +118,9 @@ func (daemon *Daemon) buildSandboxOptions(container *container.Container) ([]lib
 	for _, extraHost := range container.HostConfig.ExtraHosts {
 		// allow IPv6 addresses in extra hosts; only split on first ":"
 		parts := strings.SplitN(extraHost, ":", 2)
+		if len(parts) != 2 {
```

Comment:
Yeah, `opts.ValidateExtraHost` also validates whether the IP is valid, not only bad format.